Twig/RollbarExtension.php

```diff
@@ -89,6 +89,11 @@ public function getInitRollbarCode(array $context)
             );
         }
 
+        $rollbarJsVersion = 'v1';
```

Comment:
what if rollbar comes up with v2? then "newest" will not be the newest any more ;)